### DISCUSSION TITLE
what step need to do every time we change on akvorado.yaml and inlet.yaml?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1425] [Q&A]
Hi Vincent / expert,

I has change some info on akvorado.yaml. After change i reboot the server. But after reboot in the dashboard it still display as "ipv4-customers-1" not "ipv4-Hurricane"? Is it my step wrong? I try different browser after reboot but still same. Another thing i add some attribute at inlet.yaml "- ClassifySite("cyberjaya")" but it not reflect the info "cyberjaya". Both after change then i restart the avokado server but still not reflect.  I'm also run this command "docker compose up -d --remove-orphans" before restart server but still not reflect. Appreciate your feedback

**Before**
networks:
    103.130.12.0/22:
      name: ipv4-customers-1
      role: customers

**After**
networks:
    103.130.12.0/22:
      name: ipv4-Hurricane
      role: customers

**inlet.yaml**

core:
  workers: 6
  exporter-classifiers:
    # This is an example. This should be customized depending on how
    # your exporters are named.
    - ClassifySiteRegex(Exporter.Name, "^([^-]+)-", "$1")
    - ClassifyGroup("csf1")
    - ClassifyRegion("asia")
    - ClassifyTenant("ip core")
    - ClassifyRole("border-router")
    **- ClassifySite("cyberjaya")**

**Comment by vincentbernat** [comment]
You need to remove the `ClassifySiteRegex`. For each dimension (site, group, region), the first one wins.

**Reply by kronicklez** [reply]
Hi vincent,

If i comment "# - ClassifySiteRegex(Exporter.Name, "^([^-]+)-", "$1")" is it will effect the "ExporterName	dc5-edge2.example.com" display in the dashboard also?

![image](https://github.com/user-attachments/assets/fdaee2c4-aff4-4284-8dfd-c136155b9ff5)


Thanks and appreciate your feedback

**Reply by vincentbernat** [reply]
Yes, it will.
